packages/components/fields/creatable-select-field/src/creatable-select-field.tsx

```diff
@@ -494,6 +498,8 @@ export default class CreatableSelectField extends Component<
             createOptionPosition={this.props.createOptionPosition}
             showOptionGroupDivider={this.props.showOptionGroupDivider}
             iconLeft={this.props.iconLeft}
+            // @ts-ignore
```

Comment:
Same here.

Reply:
@ddouglasz / @CarlosCortizasCT 
That's interesting you were not getting any errors locally because I was getting this error:

`Property 'isCondensed' does not exist on type 'IntrinsicAttributes & Pick<TCreatableSelectInputProps, "id" | "horizontalConstraint" | "aria-label" | "aria-labelledby" | ... 36 more ... | "hasError"> & Partial<...> & Partial<...>'.`

And it prevented me from committing the code. I had noticed `// @ts-ignore` being used in other input components so I thought maybe adding it here was the right solution. Let me know if there's a better idea. :)

Reply:
Oh, interesting. Could be a caching issue from your local machine. perhaps restart your ide. 
I pushed the change here since I don't have that issue: https://github.com/commercetools/ui-kit/pull/2806/commits/0f64de126e00ccb3c930f32a51d4966d5d357421

Reply:
Cool! Thanks for doing that